modules/oc-mirror-imageset-config-params.adoc

```diff
@@ -269,3 +269,12 @@ channels:
 |Boolean. The default value is `false`.
 
 |===
+
+[NOTE]
+====
+In certain cases error occurs when the `minVersion` and `maxVersion` properties are used to filter for a specific version range for an operator in the `ImageSetConfiguration` parameters. The error message will state that there are `multiple channel heads`. This is because when the filter is applied, the update graph of the operator is truncated.
+
+The Operator Lifecycle Manager in OpenShift, however, expects that every operator channel contains versions that form an update graph with exactly one end point, the latest version of the operator. When applying the filter range that graph can turn into two or more separate graphs or a graph that has more than one end point.
```

Comment:
s/Operator Lifecycle Manager in OpenShift/Operator Lifecycle Manager
Remove _however_. 
s/expects/requires  Anthromorphism, per IBM Style Guide

Reply:
Ack.